Apromore-Core-Components/Apromore-Portal/src/main/webapp/themes/ap/common/img/icons/unlink.svg

```diff
@@ -1,3 +1 @@
-</svg>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" x="0" y="0" aria-hidden="true" focusable="false" role="img"><path d="M432.32,470.4,116.91,13.12A15.35,15.35,0,0,0,95.09,9.84L82.44,19.39A16.32,16.32,0,0,0,79,41.75L394.36,499a15.35,15.35,0,0,0,21.82,3.28l12.66-9.54A16.32,16.32,0,0,0,432.32,470.4Z" fill="currentColor"/><path d="M349.11,291.67c0-.24,0-.47,0-.71,0-1.54-.06-3.08-.13-4.63a129.56,129.56,0,0,0-33.44-80.88l.16-.06-4.53-4.54a129.24,129.24,0,0,0-33.45-24.18,14.89,14.89,0,0,0-6.52-1.52h0c-4,0-7.42,1-10.19,3.85l.19.09c-.13.12-.27.23-.4.36l43.93,63.68C322.19,272.78,302.78,312.71,274,348l-7.7,7.7h0l-59.75,59.76c-15.27,15.25-34.61,23.65-54.46,23.65-20.17,0-39.51-8.81-55.92-25.47l-.07-.08c-14.84-14.28-22.77-31.71-22.95-50.4C73,341.88,82.57,320,99.51,303l41.21-41.2A14.82,14.82,0,0,0,144.67,248c-2.31-9.64-2.36-14.21-2.47-23.31l0-3.15a14.67,14.67,0,0,0-14.84-14.48h0a14.66,14.66,0,0,0-10.41,4.34l-19,19.05c-9.62,9.64-22,22-38,38-50.63,50.63-50.63,133,0,183.65a129,129,0,0,0,91.82,38h0a129,129,0,0,0,91.82-38l37.94-37.9,1.87-1.87c.17-.18.35-.35.52-.52,31.8-31.75,31.91-31.89,36-36.93l.33-.42a129.15,129.15,0,0,0,28.86-77.72c0-.17,0-.34,0-.51C349.11,294.7,349.12,293.19,349.11,291.67Z" fill="currentColor"/><path d="M452.17,60.06l-.85-.85h0a128.87,128.87,0,0,0-91.08-37.33h0a129,129,0,0,0-91.72,38l-.1.11C252.38,76,240,88.35,230.33,98L215.1,113.2h0c-19,19-19.58,19.73-23,24l-.33.41c-19.9,24.47-29.5,53.47-29.05,82.2v.1c0,1.1.05,2.2.1,3.3v.07a125,125,0,0,0,1.32,13.37c0,.1,0,.19,0,.29.16,1,.33,2,.51,3l.06.29q.9,5,2.22,9.82l.06.2c.26,1,.54,1.94.83,2.91l.16.55c.27.88.55,1.75.83,2.63.06.18.12.37.19.56.31.94.63,1.88,1,2.82,0,.09.06.18.1.27.76,2.1,1.57,4.18,2.45,6.24l.22.51c.35.83.71,1.65,1.09,2.47.11.26.24.52.36.78q.51,1.11,1.05,2.22l.39.8c.39.79.79,1.56,1.19,2.34l.32.6c1,2,2.12,3.89,3.26,5.81l.44.73c.43.7.85,1.39,1.29,2.08.19.31.39.62.59.92.41.64.82,1.27,1.24,1.9l.63.93c.45.66.91,1.31,1.37,2l.55.78c1.31,1.81,2.66,3.6,4.07,5.36.22.27.45.54.67.82l1.48,1.8.82.95c.47.55.95,1.11,1.44,1.66.28.31.56.63.85.94.52.58,1.05,1.15,1.58,1.72l.76.81c.78.82,1.58,1.64,2.39,2.46h0a129.07,129.07,0,0,0,33.45,24.18,14.86,14.86,0,0,0,6.52,1.52h0c4,0,5.63.51,8.41-2.29l-.33-.14.37-.25-45.55-66A3.61,3.61,0,0,0,203,263c-9-17-5.85-35.48,0-54,6-19,19.85-32.45,38.47-50l3.09-3.09h0l52-52,12.62-12.62a71.27,71.27,0,0,1,50.68-21.09c21.34,0,42.29,9,60.61,26,35,38.39,19.86,87-9.11,116l-41.2,41.2a14.8,14.8,0,0,0-3.95,13.83c.45,1.87.84,3.46,1.2,4.88,1.72,6.95,2.31,9.29,2.44,18.38A14.67,14.67,0,0,0,384.65,305h0a14.65,14.65,0,0,0,10.4-4.35l19.11-19.12c9.61-9.62,21.95-22,37.9-37.93A129.92,129.92,0,0,0,452.17,60.06Z" fill="currentColor"/></svg>
```

Comment:
We should also update the unlink icon in Apromore-Custom-Plugins/Process-Publisher-Portal-Plugin/src/main/resources/unlink.svg